File: LibreriaV3.1/LibreriaV3.1/Negocio/TLibro.cs
using LibreriaV3._1.Comun;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LibreriaV3._1.Modelo
{
    class TLibro
    {
        public string CodLibro { get; set; }
        public string Autor { get; set; }
        public string Titulo { get; set; }
        public string Tema { get; set; }
        public string Paginas { get; set; }
        public string Precio { get; set; }
        public string Formatouno { get; set; }
        public string Formatodos { get; set; }
        public string Formatotres { get; set; }
        public string Estado { get; set; }
        public int Borrado { get; set; }

        //Constructors
        public TLibro()
        {
        }

        public TLibro(int borrado,string codLibro, string autor, string titulo, string tema, string paginas, string precio, string formatouno, string formatodos, string formatotres, string estado)
        {
            this.CodLibro = codLibro;
            this.Autor = autor;
            this.Titulo = titulo;
            this.Tema = tema;
            this.Paginas = paginas;
            this.Precio = precio;
            this.Formatouno = formatouno;
            this.Formatodos = formatodos;
            this.Formatotres = formatotres;
            this.Estado = estado;
            this.Borrado = borrado;
        }

        public TLibro(string autor, string titulo, string tema, string paginas, string precio, string formatouno, string formatodos, string formatotres, string estado)
        {
            this.CodLibro = Util.GenerarCodigo(this.GetType());
            this.Autor = autor;
            this.Titulo = titulo;
            this.Tema = tema;
            this.Paginas = paginas;
            this.Precio = precio;
            this.Formatouno = formatouno;
            this.Formatodos = formatodos;
            this.Formatotres = formatotres;
            this.Estado = estado;
            this.Borrado = 0;
        }



        //ToString
        public override string ToString()
        {
            return Titulo;
        }
    }
}

File: LibreriaV6.1/src/Negocio/TLibro.java
package Negocio;


import Persistencia.UtilSQL;

import javax.naming.NamingException;
import java.io.IOException;
import java.sql.SQLException;

public class TLibro {

	private String codLibro;
	private String autor;
	private String titulo;
	private String tema;
	private String paginas;
	private String precio;
	private String formatouno;
	private String formatodos;
	private String formatotres;
	private String estado;


	public TLibro() {
	}

	public TLibro(String titulo, String autor, String tema, String paginas, String precio, String f1, String f2, String f3, String estado) throws ClassNotFoundException, SQLException, IOException, NamingException {
		super();
		this.codLibro= UtilSQL.generarCodigo(this.getClass());
		this.titulo = titulo;
		this.autor = autor;
		this.tema = tema;
		this.paginas = paginas;
		this.precio = precio;
		this.formatouno = f1;
		this.formatodos = f2;
		this.formatotres = f3;
		this.estado = estado;
	}

	public TLibro(String codLibro, String titulo, String autor, String tema, String paginas, String precio, String f1, String f2, String f3, String estado) {
		super();
		this.codLibro=codLibro;
		this.titulo = titulo;
		this.autor = autor;
		this.tema = tema;
		this.paginas = paginas;
        this.precio = precio;
		this.formatouno = f1;
		this.formatodos = f2;
		this.formatotres = f3;
		this.estado = estado;
	}



	public String getAutor() {
		return autor;
	}

	public void setAutor(String autor) {
		this.autor = autor;
	}

	public String getTitulo() {
		return titulo;
	}

	public void setTitulo(String titulo) {
		this.titulo = titulo;
	}

	public String getTema() {
		return tema;
	}

	public void setTema(String tema) {
		this.tema = tema;
	}

	public String getPaginas() {
		return paginas;
	}

	public void setPaginas(String paginas) {
		this.paginas = paginas;
	}

    public String getPrecio() {
        return precio;
    }

    public void setPrecio(String precio) {
        this.precio = precio;
    }

    public String getFormatouno() {
		return formatouno;
	}

	public void setFormatouno(String formatouno) {
		this.formatouno = formatouno;
	}

	public String getFormatodos() {
		return formatodos;
	}

	public void setFormatodos(String formatodos) {
		this.formatodos = formatodos;
	}

	public String getFormatotres() {
		return formatotres;
	}

	public void setFormatotres(String formatotres) {
		this.formatotres = formatotres;
	}

	public String getEstado() {
		return estado;
	}

	public void setEstado( String estado) {
		this.estado= estado;
	}

	public String getCodLibro() {
		return codLibro;
	}

	public void setCodLibro(String codLibro) {
		this.codLibro = codLibro;
	}

	@Override
	public String toString() {
		return titulo;
	}

}
